.github/dependabot.yml

```diff
@@ -42,3 +42,14 @@ updates:
     commit-message:
       prefix: "[Dependabot-automated] "
     open-pull-requests-limit: 6
+
+  # mlrun pipelines adapters
+  - package-ecosystem: pip
+    directory: /
+    schedule:
+      interval: daily
+    allow:
+      - dependency-name: "mlrun-pipelines-kfp*"
```

Comment:
Regex is allowed? Also nicely done!

Reply:
followed the example here https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#allow